api/lib/infrastructure/repositories/cpf-certification-result-repository.js

```diff
@@ -24,47 +28,48 @@ const findByBatchId = async function (batchId) {
       ) ORDER BY "competence-marks"."competence_code" asc) as "competenceMarks"`),
     )
     .innerJoin('competence-marks', 'competence-marks.assessmentResultId', 'assessment-results.id')
-    .where('certification-courses.cpfFilename', batchId)
+    .where('compte-personnel-formation.filename', batchId)
     .groupBy('certification-courses.id', 'assessment-results.pixScore', 'sessions.publishedAt');
   return cpfCertificationResults.map((certificationCourse) => new CpfCertificationResult(certificationCourse));
 };
 
 const markCertificationCoursesAsExported = async function ({ certificationCourseIds, filename }) {
-  const now = new Date();
-
-  return knex('certification-courses')
-    .update({ cpfFilename: filename, cpfImportStatus: cpfImportStatus.READY_TO_SEND, updatedAt: now })
-    .whereIn('id', certificationCourseIds);
+  return knex('compte-personnel-formation')
+    .update({ filename: filename, importStatus: cpfImportStatus.READY_TO_SEND, updatedAt: knex.fn.now() })
+    .whereIn('certificationCourseId', certificationCourseIds);
 };
 
 const markCertificationToExport = async function ({ startDate, endDate, limit, offset, batchId }) {
-  const now = new Date();
-
-  return knex
-    .with('certification-courses-to-mark', (qb) => {
-      const query = _selectCpfCertificationResults(qb);
-      return _filterQuery(query, startDate, endDate)
-        .select('certification-courses.id')
-        .where((qb) => {
-          qb.orWhereNull('certification-courses.cpfImportStatus');
-          qb.orWhere('certification-courses.cpfImportStatus', cpfImportStatus.PENDING);
-        })
+  return await knex
+    .into(
+      knex.raw('?? (??, ??, ??, ??, ??)', [
```

Comment:
Je ne comprends pas ```knex.raw('?? (??, ??, ??, ??, ??)'```

Reply:
C'est le formalisme utilisé par knex pour faire une [Requête préparée](https://fr.wikipedia.org/wiki/Injection_SQL#Utilisation_d'une_requ%C3%AAte_pr%C3%A9par%C3%A9e)

Plus précisement ici c'est le formalisme pour les identifiants de colonne : https://knexjs.org/guide/raw.html#raw-parameter-binding

Reply:
Vu que c'est en dur, pourquoi ne pas les mettre directement?
```suggestion
return await knex
    .into(
      knex.raw('table (colonne1, colonne2)', [
```